Header header
string output_weight_file_name
string dmp_name
geometry_msgs/Pose[] poses
int64 n_bfs
int64 n_dmps
---
string result